mtgjson5/providers/tcgplayer.py

```diff
@@ -276,11 +276,33 @@ def generate_mtgjson_sealed_product_objects(
 
         mtgjson_sealed_products = []
 
+        # Load cache if not loaded
+        with constants.RESOURCE_PATH.joinpath("booster_box_size_overrides.json").open(encoding="utf-8") as f:
+            booster_box_size_overrides = json.load(f)
+
+        code = self.code.upper()
```

Comment:
There is no `self.code` in this class, you'll need to pass `set_code: str` in to the function